recipes/yarr/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "yarr" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/yarr/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 56dd0a42f5260470239f76b89aad016913e8267fbf3e10227abf1797618de18d
+  patches:
+    - patches/0001-yarr_873.patch
+    - patches/0002-patch-strip-debug.patch
+    - patches/0003-patch-dependency-resolution-for-osx.patch
+
+build:
+  skip: true  # [win]
+  script:
+    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -S "${SRC_DIR}" -B build  # [osx]
+    - CXXFLAGS="${CXXFLAGS}" cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -S "${SRC_DIR}" -B build  # [linux]
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+    - mkdir -p $PREFIX/share/yarr
+    - rsync -ravz $SRC_DIR/configs $PREFIX/share/yarr
+    - find $PREFIX/share/yarr/configs -name '*.json' -type f -print -exec sed -i "s@\"configs/@\"$PREFIX/share/yarr/configs/@g" {} +
+    - cat $PREFIX/share/yarr/configs/controller/emuCfg_rd53a.json
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+    - rsync
+    - sed
+  host: []
+  run: []
```

Comment:
```suggestion
```

Not wrong as in, but you can drop these keys if they're empty. I find that cleaner visually, but again, not wrong as is.